libs/tokens/src/const/tokensList.json

```diff
@@ -0,0 +1,93 @@
+{
+  "1": [
+    {
+      "id": "hc_zMahDEXc_WMKgPAUpK",
```

Comment:
`nanoid`, I used it in `addListAtom`

Reply:
Right, but are they fixed or somehow related to the content?

Reply:
But why do we need an id? 
Wouldn't this be a problem if we update this id, it could make a duplicate. Right?

Why not using, either:
- the URL as ID of the list
- Or probably better, a logical name, like `cowswap`, `coingecko`, etc.

Reply:
You are right. It was a part of overengineered version.
Refactored and simplified it: https://github.com/cowprotocol/cowswap/pull/3201/commits/b12a42962a6dfa7c3698e336143d45278381d0da